pkg/spoke/managedcluster/claim_controller.go

```diff
@@ -82,7 +85,9 @@ func (c managedClusterClaimController) exposeClaims(ctx context.Context, syncCtx
 	managedCluster *clusterv1.ManagedCluster) error {
 	reservedClaims := []clusterv1.ManagedClusterClaim{}
 	customClaims := []clusterv1.ManagedClusterClaim{}
-	clusterClaims, err := c.claimLister.List(labels.Everything())
+	requirement, _ := labels.NewRequirement(labelCustomizedOnly, selection.DoesNotExist, []string{})
```

Comment:
add a comments here

Reply:
done

Reply:
let's say lables of clusterclaim with spoke only will not be synced to managedcluster.Status at hub

Reply:
corrected it.